### ISSUE TITLE
Create easy changeable parameters

### ISSUE THREAD

**Issue by juanabascal** [juanabascal/tf-flower-classificator#7]
We should add tf.Flags for easily changing parameters like:

- [x] Learning rate
- [x] Image input size
- [x] Number of images in the training dataset

If you think out more tf.Flags we should add add them to the list.

**Comment by dgdelahera** [comment]
Done!
